src/main/java/com/google/devtools/build/lib/rules/java/JavaToolchainTool.java

```diff
@@ -64,8 +64,8 @@ static JavaToolchainTool fromStarlark(@Nullable StructImpl struct) throws RuleEr
   public abstract NestedSet<Artifact> data();
 
   /**
-   * JVM flags to invoke the tool with, or empty if it is not a {@code _deploy.jar}. Location
-   * expansion is performed on these flags using the inputs in {@link #data}.
+   * JVM flags to invoke the tool with. Location expansion is performed on these flags using the
```

Comment:
I adjusted this comment.

@hvadehra I think that the second sentence is outdated since the Starlarkified `java_toolchain` rule performs location expansion *before* a `JavaToolchainTool` instance is created.